swap/src/cli.rs

```diff
@@ -24,6 +25,9 @@ pub enum Command {
 
         #[structopt(long = "receive-btc", help = "Bitcoin amount as floating point nr without denomination (e.g. 1.25)", parse(try_from_str = parse_btc))]
         receive_bitcoin: bitcoin::Amount,
+
+        #[structopt(flatten)]
+        config: Config,
```

Comment:
What could be interesting is pass this parameter **before** the command so it's available for all commands without the repetition. E.g. : `swap --config foo/bar.toml buy-xmr`

Reply:
The only globally available parameter is `--datadir`, because it is valid for for all commands, including `history`, but `--config` is only relevant for swap execution. 

Note that `swap` is currently not a `Command`, but just the name of the binary.

Reply:
> Note that swap is currently not a Command, but just the name of the binary.

Yes I know, is my example incorrect?